ambry-api/src/main/java/com/github/ambry/store/MessageInfo.java

```diff
@@ -439,6 +453,7 @@ public Builder isDeleted(boolean isDeleted) {
      * @param isTtlUpdated the isTtlUpdated to set.
      * @return This builder.
      */
+    @JsonProperty("ttlUpdated")
```

Comment:
The code changes look generally ok to me. But do we think that serializing and deserializing in JSON format is the optimal thing to do?
Currently, we are serializing only values and using the serialization order to deserialize. Using JSON as the serialization and deserialization format will add the field names as the key in the serialized format. These field names will be strings in all cases. This will lead to larger serialized data, and potentially need larger network bandwidth to transfer the same amount of information.

Reply:
this isn't going on critical read and write path. The purpose of this is for an admin command to show message info of a given blob id in each host.